modular_pariah/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -6,8 +6,11 @@
 	metabolization_rate = 1.75 * REAGENTS_METABOLISM
 
 /datum/reagent/pax/catnip/on_mob_life(mob/living/carbon/M)
-	if(prob(20))
-		M.emote("nya")
-	if(prob(20))
-		to_chat(M, span_notice("[pick("Headpats feel nice.", "The feeling of a hairball...", "Backrubs would be nice.", "Mew")]"))
+	if(isfelinid(M))
+		if(prob(20))
+			M.emote("nya")
+		if(prob(20))
+			to_chat(M, span_notice("[pick("Headpats feel nice.", "Backrubs would be nice.", "Mew")]"))
+	else
+		to_chat(M, span_notice("[pick("I feel oddly calm.", "I feel relaxed.", "Mew?")]"))
```

Comment:
Not feeling this drink, feels off in tone to me.

Reply:
name?

Reply:
cause the catnip was already in

Reply:
The drink itself was Skyrat specific, I just feel like the whole drink itself is weirdly out of tone for Pariah.